### DISCUSSION TITLE
Option to hide update button?

### DISCUSSION THREAD

**Discussion by edikanabia** [nathanhoad/godot_dialogue_manager#494] [Ideas]
Even though I think it's really cool that the add-on has a feature to update from within the engine editor, I am still feeling the sting of upgrading to the most recent version and having all of my functions break with no clear direction on how to fix it. I feel like I've spent enough time finagling with the add-on and would like to stay on the current version but I find the bright green indicator distracting. I know it's kind of a nothing issue, but still.

**Comment by nathanhoad** [comment]
I've [added an option](https://github.com/nathanhoad/godot_dialogue_manager/pull/496) to turn off auto-update that will be in the next release (ironically needing an update to get it).
